src/Core/Asset/HandleToSkeleton.cpp

```diff
@@ -17,18 +17,16 @@ namespace {
 void addBone( const int parent,                        // index of parent bone
               const uint dataID,                       // index in map
               const Ra::Core::Asset::HandleData& data, // handle data
-              const Ra::Core::AlignedStdVector<Ra::Core::Vector2i>& edgeList, // list of edges
+              const Ra::Core::AlignedStdVector<Ra::Core::Vector2ui>& edgeList, // list of edges
               std::vector<bool>& processed,        // which ids have been processed
               Core::Animation::Skeleton& skelOut ) // correspondance between bone name and bone idx
 {
     if ( !processed[dataID] )
     {
         processed[dataID] = true;
         const auto& dd    = data.getComponentData()[dataID];
-        uint index        = skelOut.addBone( parent,
-                                      data.getFrame() * dd.m_offset.inverse(),
-                                      Ra::Core::Animation::Handle::SpaceType::MODEL,
-                                      dd.m_name );
+        int index         = skelOut.addBone(
```

Comment:
Can `index` be negative ? 
Why using `int` and not `uint`, or `Ra::Core::Index` ?